src/_includes/assets/js/app.js

```diff
@@ -1,4 +1,28 @@
 console.log(
   '%cA black belt is a white belt that didn\'t quit.',
-  'color: #fff; font-size: 80px; text-shadow: 1px 1px 1px #000; font-family: serif;'
+  'color: #fff; font-size: 20px; text-shadow: 1px 1px 1px #000; font-family: serif;'
 )
+
+var navToggle = document.getElementById('navigation-toggle')
+var nav = document.getElementById('navigation')
+
+navToggle.addEventListener('click', function() {
+  if (nav.classList.contains('open')) {
+    nav.classList.remove('open')
+  } else {
+    nav.classList.add('open')
+  }
+})
+
+window.addEventListener('resize', function() {
+  if (document.width > 960) {
```

Comment:
can you explain what this is doing? Is there any reason you can't fix this with a media query (width > 960, just make the thing hidden?)

Reply:
Good question! Because there are CSS `transition`s on the menu as it shows/hides in mobile view, browsers honor those transitions and will visibly "zoom" the menu out when a user shrinks their browser below 960. This temporary add/remove class function catches that resize and prevents _any_ transitions from firing at the breakpoint where the nav layout changes.

I can take another look tonight at trying to solve that same issue w/ just CSS.

Reply:
no, that's fine.  that makes sense. leave it as is.